package-lock.json

```diff
@@ -5,6 +5,7 @@
 	"requires": true,
 	"packages": {
 		"": {
+			"name": "amazon-states-language-service",
 			"version": "1.7.0",
```

Comment:
package-log.json's version is still 1.7.0. `npm version patch` should update both files

Reply:
i should* have run yarn test, but reverted and ran `npm version patch` to let the tooling do it's thing.
good catch.